src/client/Client.ts

```diff
@@ -60,30 +60,30 @@ export class Client {
   }
 
   /**
-   * get epoch id(or epoch height as a hex string)
+   * get block height as a hex string
    */
-  public async getLatestEpochId(): Promise<string> {
-    const res = await this.rawClient.getEpoch();
-    return res.getEpoch.header.epochId;
+  public async getLatestHeight(): Promise<string> {
```

Comment:
muta has a [breaking change](https://github.com/nervosnetwork/muta/pull/131) that changes "id" to "height", "epoch" to "height", So recommend this way`height -> number`, `id -> height` here.

the function signature would be `getLatestBlockHeight` and `getBlockNumber`